src/main/java/org/folio/circulation/domain/UpdateRequestQueue.java

```diff
@@ -69,19 +71,28 @@ public static UpdateRequestQueue using(Clients clients) {
         firstRequest.changeStatus(RequestStatus.OPEN_AWAITING_PICKUP);
 
         if (firstRequest.getHoldShelfExpirationDate() == null) {
-          return servicePointRepository.getServicePointById(requestPickupServicePointId)
+          return servicePointRepository.getServicePointWithTenantTimeZone(requestPickupServicePointId)
```

Comment:
I don't think I understand this change. Why not fetch the service point and timezone information separately, then combine them here, rather than making the service point be aware of the timezone?